frontend/src/store/sliceSettingsStore.js

```diff
@@ -122,6 +122,7 @@ const mutations = {
       internal_solid_infill_pattern: 'monotonic',
       brim_type: 'auto_brim',
      };
+     state.profileOverwrites = { ...state.profileOverwrites, ...profilePreset }
```

Comment:
Good catch. But I think we should send profileOverwrites to {}.

Reply:
I think this is needed for filling the inputs of printer profile preset. This is for the new way of getting/setting the preset data.